test-resources/eastwood_warning.clj

```diff
@@ -5,3 +5,22 @@
 (def reflection-warning
   (letfn [(get-path [x] (.getPath x))]
     (get-path (java.io.File. "a.clj"))))
+
+(def ^:dynamic *dynamic*)
+
+(defn no-pre-post-warning--1
+  "This function should not return a prepost warning (because the precondition is dynamic)"
+  [x]
+  {:pre [*dynamic*]}
+  x)
+
+(defn no-pre-post-warning--2
```

Comment:
I managed to break the tests with this new case, can you take a look?

(attempted to fix myself w/ the `& vs` change below, no luck)

Reply:
there should be 1 warning in this case, because `x` is logically always true.

I'll take a look

Reply:
ah yes, the test case I meant to write was more like: `(defn foo [x] {:pre [x]} ...)`

* :pre refers to the arg 
* variations: ` {:pre [x *foo*]}`, ` {:pre [*foo* x]}` so that we exercise cardinality